VideoWeb/VideoWeb/ClientApp/src/app/services/api/quick-links.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { ApiClient, QuickLinkParticipantJoinRequest, QuickLinkParticipantJoinResponse, Role } from 'src/app/services/clients/api-client';
+import { Observable } from 'rxjs';
+import { filter, mergeMap, take, tap } from 'rxjs/operators';
+import { SecurityConfigSetupService } from 'src/app/security/security-config-setup.service';
+import { IdpProviders } from 'src/app/security/idp-providers';
+import { SecurityServiceProvider } from 'src/app/security/authentication/security-provider.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class QuickLinksService {
+    constructor(
+        private apiClient: ApiClient,
+        private securityConfigSetupService: SecurityConfigSetupService,
+        private securityServiceProviderService: SecurityServiceProvider
+    ) {}
+
+    getQuickLinkParticipantRoles(): Observable<Role[]> {
+        return this.apiClient.getQuickLinkParticipantRoles();
+    }
+
+    validateQuickLink(hearingId: string): Observable<boolean> {
+        return this.apiClient.validateQuickLink(hearingId);
+    }
+
+    joinHearing(hearingId: string, name: string, role: Role): Observable<boolean> {
```

Comment:
This should probably be joinConference and have conferenceId not hearingId

Reply:
Actually hearingId is what is used, but it is still the conference that is being joined